elements/noflo-account.html

```diff
@@ -228,13 +228,28 @@
           </div>
         </div>
       </template>
+      <template if="{{ plan === 'free' && !askForScope.length }}">
+        <div class="banner">
+          You're using a free Flowhub plan. Upgrading to Pro gives you access to your private projects on GitHub.
```

Comment:
Propose tweaking the text, shorter and more active language: "... Upgrade to Pro to access your private projects on GitHub."